### PR TITLE
feat: add endpoint to get course exams

### PR DESCRIPTION
**Pull request by alangsto**
**JIRA:** [MST-1890](https://2u-internal.atlassian.net/browse/MST-1890)

**Description:** Add an endpoint to retrieve all active exams for a given course_id

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: edx_exams/apps/router/views.py [hunk 1]
```diff
@@ -40,11 +41,21 @@ def patch(self, request, course_id):
                 exam['is_proctored'] = exam_type.is_proctored
                 exam['is_practice_exam'] = exam_type.is_practice
 
-        response_data, status = register_exams(course_id, exam_list)
+        response_data, response_status = register_exams(course_id, exam_list)
```
Comment (alangsto) [inline edx_exams/apps/router/views.py:44]: renamed variable after importing `status` from rest_framework

File: edx_exams/apps/router/views.py [hunk 2]
```diff
@@ -40,11 +41,21 @@ def patch(self, request, course_id):
...
 
         return JsonResponse(
             data=response_data,
-            status=status,
+            status=response_status,
+            safe=False,
+        )
+
+    def get(self, request, course_id):  # pylint: disable=unused-argument
```
Comment (alangsto) [inline edx_exams/apps/router/views.py:52]: Needed to add corresponding view in the middleware. Left this as an essentially empty response for now, but can also add support for the legacy view if needed.

Comment (Zacharis278) [reply edx_exams/apps/router/views.py:52]: interesting, good call that we needed this.

**Review by Zacharis278 — APPROVED** [review]
👍

**Pull request merged**